ui/src/routes/group-chat/[chatId]/+page.svelte

```diff
@@ -38,7 +40,7 @@
 
 <div class="top-bar" style="gap: 0">
 	<wa-button class="circle" appearance="plain" href="/">
-		<wa-icon src={wrapPathInSvg(mdiArrowLeft)}> </wa-icon>
+		<wa-icon src={wrapPathInSvg(mdiArrowBack)}> </wa-icon>
```

Comment:
Curious if this is related to localization? Is this a thing where in RTL languages the arrow points right? Or just a different icon, unrelated?

Reply:
Yes exactly. `mdiArrowBack` is defined in utils as left or right depending on the direction of the document.